config/locales/en.yml

```diff
@@ -33,6 +33,18 @@ en:
       invalid_email: You must enter the email address that matches this account.
       alert: 'This will delete all of your account data, applications, and history.  Proceed with caution!'
       apps_deletion_warning: 'The following applications will also be deleted:'
+  devise:
+    welcome:
+      line_1: Welcome to MyUSA from %{app_name}
+      line_2: Create a MyUSA account or log in to your existing account to access %{app_name}.
+    connect_with: Connect with MyUSA
+    enter_your_email:  Enter your email to log in
```

Comment:
Not to be anal-retentive, but are we doing two spaces after periods, or one? (.e.g before "proceed with caution.")